tools/cli/README.md

```diff
@@ -28,12 +28,12 @@ In this directory:
 Local
 
 ```bash
-cargo run deploy-contracts
+cargo run deploy contracts
```

Comment:
Not here but above… the link to layer-sdk is a price repo and we should copy those instructions into this file

Was wondering about docs on how to create the wallet and use the basic wallet and faucet commands. Good to have this cleanly documented, as it will need to be on docs-avs as well

Reply:
updated with direct commands in https://github.com/Lay3rLabs/avs-toolkit/pull/38/commits/c17f40398c786ab2c7b812c04e5e69d971e3868b